### ISSUE TITLE
FXSound Not Opening On Top Of Other Windows When Opening By Clicking SysTray App

### ISSUE THREAD

**Issue by JustEnf** [fxsound2/fxsound-app#195]
**Describe the bug**
When opening FXSound UI from the FXSound SysTray app the UI window opens behind other windows instead of opening on top of other program windows.

**To Reproduce**
1. Open Chrome or another application so it has focus
2. Click on FXSound SysTray app
3. Observe that FXSound UI opens behind other application windows, most (90%) of the time, instead of on top

**Expected behavior**
FXSound SysTray app should open the FXSound UI on top of other open application windows.

**System information**
 - Windows version 11 24H2 or 23H2
 - Audio device - Dell/Realtek built in sound
 - FXSound version v1.1.27

**Comment by JustEnf** [comment]
Also, If the UI pops up on top as expected the first time from the systray app then click on the systray app again to minimize the UI then click the systray app one more time (to open UI again) you should see the UI is now opening behind other windows

**Comment by ReinhardC** [comment]
Yes clicking the tray icon should have the same effect as clicking the X button

**Comment by JustEnf** [comment]
Yes I understand that. The issue I described has to do with the application
window opening behind all other windows instead of on top, when it should
be the current active window.

**Comment by JustEnf** [comment]
I understand how the x works and how your app should close. I'm not talking
about it closing. I'm talking about it opening the first time. When you
click on the stray, to 'open' the application, the application opens but it
is behind all other windows. Why would the window like this be opened and
then displayed behind of all other windows? That makes me go to the taskbar
to click on your application icon to make it come to the forward position
so I can see it.

**Comment by ReinhardC** [comment]
Ah the misunderstanding is around the two meanings of the word "should".  ("It should be ok" vs. "we really should go now")

I'm on the same page, being annoyed by that bug too. To fix it, what would have to be done, is use the code behind the "X" button for the tray icon in maximized mode - that's what I meant to say.

**Comment by bvijay74** [comment]
This issue is fixed in version 1.1.29.0
